src/LocaleProvider.tsx

```diff
@@ -0,0 +1,120 @@
+import * as RNLocalize from 'react-native-localize';
+import React, {createContext, useContext, useEffect, useState} from 'react';
+import {Language} from '@atb/translations';
+import {usePreferences} from '@atb/preferences';
+import {
+  DEFAULT_LANGUAGE,
+  DEFAULT_REGION,
+  FALLBACK_LANGUAGE,
+} from '@atb/translations/commons';
+
+export type Locale = {
+  language: Language;
+  region: string;
+  localeString: string;
+};
+
+type LocaleState = {
+  locale: Locale;
+};
+
+const norwegianLocale: Locale = {
+  language: Language.Norwegian,
+  region: 'NO',
+  localeString: 'nb_NO',
+};
+
+const fallbackLocale: Locale = {
+  language: FALLBACK_LANGUAGE,
+  region: DEFAULT_REGION,
+  localeString: `${FALLBACK_LANGUAGE}_${DEFAULT_REGION}`,
+};
+
+export const LocaleContext = createContext<LocaleState>({
+  locale: norwegianLocale,
+});
+
+/**
+ * We'll create a locale as a mix of system locale and language preferences set in 'my AtB'
+ * where we always use region from system locale
+ * and language from settings or from system locale.
+ * if system locale language is not supported by the app we fallback to FALLBACK_LANGUAGE
+ */
+const LocaleContextProvider: React.FC = ({children}) => {
+  const locale = useLocale();
+  return (
+    <LocaleContext.Provider value={{locale}}>{children}</LocaleContext.Provider>
```

Comment:
Trenger vi å nest'e her? I de fleste andre contexts så sender vi bare state inn uten å legge det under et eget felt.

Så det blir `const locale = useLocaleContext()` i stedet

```suggestion
    <LocaleContext.Provider value={locale}>{children}</LocaleContext.Provider>
```

Reply:
godt mulig. skal teste.

Reply:
ja, funker fint. krevde en liten omskriving, men ble samtidig kvitt litt unødvendig wrapping :+1: